src/feedlist_formaction.cpp

```diff
@@ -162,7 +162,9 @@ void feedlist_formaction::process_operation(operation op, bool automatic, std::v
 			if (feed) {
 				if (feed->rssurl().substr(0,6) != "query:") {
 					LOG(level::INFO, "feedlist_formaction: opening feed at position `%s': %s", feedpos, feed->link());
-					v->open_in_browser(feed->link());
+					if (!v->open_in_browser(feed->link()))
+						v->show_error(_("Browser failed to open the link!"));
```

Comment:
Please use curly brackets even when the body is a single statement. This also applies to a number of edits below.